app/collections/show-list.cjsx

```diff
@@ -32,14 +32,24 @@ SubjectNode = createReactClass
     project: null
 
   componentWillMount: ->
-    @fetchProject(@props.subject)
-      .then (project) =>
-        @setState {project}
-        @isFavorite(project)
+    # if the project context is set use that
+    if @props.projectContext
```

Comment:
I can't see this prop being passed into this component anywhere else in the code.

Reply:
https://github.com/zooniverse/Panoptes-Front-End/blob/3bf1d118038a750283a59e5cf515f4808d01663f/app/collections/show-list.cjsx#L277

Reply:
Nevermind, I think I was looking at the wrong component. This is a prop of the subject viewer, not the list itself?